backend/api/routes/bundles.py

```
@@ -0,0 +1,35 @@
+from backend.api.auth import CurrentUser, require_current_user
+from backend.api.schemas import BundleRun, ActionApprovalRequest
+from backend.services.rag_service import get_rag_application
+
+router = APIRouter()
+
+
+@router.post("/bundles/{run_id}/approve", response_model=BundleRun)
+def approve_bundle_step(
+    run_id: str,
+    request: ActionApprovalRequest,
+    current_user: CurrentUser = Depends(require_current_user)
```

Ruff (0.15.9)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)